CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log - @itwin/insights-client
 
+## 0.3.4
+Fri, 25 Nov 2022
+
+### Patches
+
+- Added missing exports.
```

Comment:
Missing exports for what? Which clients are affected?

Reply:
changed to specify that base paths are being exported